docs/source/conf.py

```diff
@@ -62,10 +62,22 @@
     ("py:obj", "trio._abc.SendType"),
     ("py:obj", "trio._abc.T"),
     ("py:obj", "trio._abc.T_resource"),
+    ("py:class", "types.FrameType"),
 ]
 autodoc_inherit_docstrings = False
 default_role = "obj"
 
+# These have incorrect __module__ set in stdlib and give the error
+# `py:class reference target not found`
+# Some of the nitpick_ignore's above can probably be fixed with this.
+# See https://github.com/sphinx-doc/sphinx/issues/8315#issuecomment-751335798
+autodoc_type_aliases = {
+    # aliasing doesn't actually fix the warning for types.FrameType, but displaying
+    # "types.FrameType" is more helpfun than just "frame"
```

Comment:
```suggestion
    # "types.FrameType" is more helpful than just "frame"
```

idea for a future PR, we might find `codespell` useful?

Reply:
I'm a +1 on codespell, I've heard good things about it!

Reply:
no fun allowed.

Yeah `codespell` has caught a bunch of typos of mine in flake8-trio, I'm all for adding it.